timeseries/src/autogluon/timeseries/dataset/ts_dataframe.py

```diff
@@ -1008,3 +1008,7 @@ def __dir__(self) -> List[str]:
         # This hides method from IPython autocomplete, but not VSCode autocomplete
         deprecated = ["get_reindexed_view", "to_regular_index"]
         return [d for d in super().__dir__() if d not in deprecated]
+
+    def to_data_frame(self) -> pd.DataFrame:
```

Comment:
```suggestion
    def to_pandas(self) -> pd.DataFrame:
```

I'd rename this method since `TimeSeriesDataFrame` is a `DataFrame`.

Reply:
I'd prefer to keep the current name to be consistent with `from_data_frame()`